src/features/import/utils/problems/predictProblems.spec.ts

```diff
@@ -18,6 +18,16 @@ function makeFullField(
 ): ZetkinCustomField {
   return {
     description: '',
+    enum_choices: [
+      {
+        key: 'first',
+        label: 'First',
+      },
+      {
+        key: 'second',
+        label: 'Second',
+      },
+    ],
```

Comment:
Why does a field of type `CUSTOM_FIELD_TYPE.TEXT` have `enum_choices`? Isn't this a piece of mock data that will never occur in real life, and can therefore result in false negatives or even false positives?

Reply:
You're right, this has been removed.